buckets/ssl_buckets.c

```diff
@@ -1788,6 +1821,9 @@ static serf_ssl_context_t *ssl_init_context(serf_bucket_alloc_t *allocator)
     ssl_ctx->protocol_callback = NULL;
     ssl_ctx->protocol_userdata = NULL;
 
+    ssl_ctx->error_callback = NULL;
+    ssl_ctx->error_userdata = NULL;
```

Comment:
We call those batons, so: `error_baton`. Hmph, should do the same for `protocol_userdata` ⟶ `protocol_baton`.